server/wfnews-service-api/src/main/java/ca/bc/gov/nrs/wfnews/service/api/v1/impl/IncidentsServiceImpl.java

```diff
@@ -0,0 +1,213 @@
+package ca.bc.gov.nrs.wfnews.service.api.v1.impl;
+
+import com.mashape.unirest.http.HttpResponse;
+import com.mashape.unirest.http.JsonNode;
+import com.mashape.unirest.http.Unirest;
+
+import ca.bc.gov.nrs.wfnews.service.api.v1.IncidentsService;
+import ca.bc.gov.nrs.wfnews.api.rest.v1.resource.IncidentResource;
+import ca.bc.gov.nrs.wfnews.api.rest.v1.resource.IncidentListResource;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+import org.json.JSONObject;
+import org.json.JSONArray;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+
+public class IncidentsServiceImpl implements IncidentsService{
+
+    private static final Logger logger = LoggerFactory.getLogger(IncidentsServiceImpl.class);
+      
+    @Value("${wfnews-agol-query.url}")
+    private String agolQueryUrl;
+
+    @Override
+    public IncidentListResource getIncidents(String status, String date, Double minLatitude, Double maxLatitude, Double minLongitude, Double maxLongitude){
+    	IncidentListResource result = new IncidentListResource();
+    	
+        try{
+        	
+            HttpResponse<JsonNode> response = Unirest
+            .post(agolQueryUrl)
+            .header("Content-Type", "application/json")
+            .header("Accept", "*/*")
+            .asJson(); 
+            
+            if (response != null) {
+            	result = getIncidentResourceListFromJsonBody(response.getBody());
```

Comment:
Ideally the filtering is left to the AGOL service to limit the amount of data sending back and forth between the services.